sw/device/lib/dif/dif_otbn.h

```diff
@@ -255,6 +256,17 @@ dif_otbn_result_t dif_otbn_is_busy(const dif_otbn_t *otbn, bool *busy);
 dif_otbn_result_t dif_otbn_get_err_bits(const dif_otbn_t *otbn,
                                         dif_otbn_err_bits_t *err_bits);
 
+/**
+ * Get the instruction counter value.
+ *
+ * @param otbn OTBN instance
+ * @param[out] insn_cnt The number of instructions executed by the OTBN.
+ * @return `kDifOtbnBadArg` if `otbn` or `err_code` is `NULL`,
+ *         `kDifOtbnOk` otherwise.
+ */
+dif_otbn_result_t dif_otbn_get_insn_cnt(const dif_otbn_t *otbn,
```

Comment:
Please avoid unnecessary initialisms. `instruction_count` is a perfectly cromulent function name.

Reply:
The current name matches the register name in hardware and the documentation there. I find this consistency more valuable (for a low-level DIF) than a longer name.